gulpfile.js

```diff
@@ -21,6 +21,8 @@ var pkg = require('./package.json'),
   uglify = require('gulp-uglify'),
   isDist = process.argv.indexOf('deploy') >= 0;
 
+var MAX_HTML_FILE_SIZE = 2000; // kbytes
```

Comment:
I think should should go higher. Let's make the default 100MB. No one is really going to try to pass around a presentation file bigger than that.